### PR TITLE
[DM-2442] [GRAFT] opcua changelog

### PR DESCRIPTION
**Pull request by jakubsobolewskisag**
opcua device types encoding bugfix

### CODE REVIEW

File: content/change-logs/device-management/cumulocity-10-18-529-0-opcua-fix-device-type-encoding.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: 
+title: Fixed special characters encoding in OPC-UA device types
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-18-529-0-opcua-fix-device-type-encoding.md]:
```suggestion
title: Fixed special characters encoding in OPC UA device types
```

File: content/change-logs/device-management/cumulocity-10-18-529-0-opcua-fix-device-type-encoding.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-MLn0oFRX-
+    label: opcua
+ticket: DM-2442
+version: 10.18.529.0
+---
+When OPC-UA device type was created with special characters in any of the fields, the gateway was not reading it correctly which caused an error while creating mappings. This problem is now fixed - gateway encodes and decodes special characters as expected.
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-18-529-0-opcua-fix-device-type-encoding.md]:
```suggestion
Previously, when creating an OPC UA device type with special characters in any of the fields, the gateway was not reading it correctly which caused an error while creating mappings. This issue is now fixed, the gateway encodes and decodes special characters as expected.  
```

File: content/change-logs/device-management/cumulocity-10-18-529-0-opcua-fix-device-type-encoding.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+product_area: Device management & connectivity
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-MLn0oFRX-
+    label: opcua
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-18-529-0-opcua-fix-device-type-encoding.md]: I assume that for 2024 the build artifact still is cumulocity?

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/device-management/cumulocity-10-18-529-0-opcua-fix-device-type-encoding.md (hunk 3 above)
Comment (jakubsobolewskisag) [reply content/change-logs/device-management/cumulocity-10-18-529-0-opcua-fix-device-type-encoding.md]: oops... my bad! Good catch :)

**Review by aaronraab — APPROVED** [review]

**Pull request merged**